src/StructureMap.DynamicInterception/MethodInvocation.cs

```diff
@@ -39,6 +54,21 @@ public MethodInfo MethodInfo
             get { return _invocation.Method; }
         }
 
+        public MethodInfo MethodInfoInvocationTarget
+        {
+            get { return _invocation.MethodInvocationTarget; }
+        }
+
+        public object Proxy
```

Comment:
It is occasionally useful to know if there are other proxies being invoked in the chain. But this is an edge case, so i've removed it.